src/hashmap.rs

```diff
@@ -340,7 +340,7 @@ mod tests {
     // Unfortunately the order of the keys can change. Doesn't seem to make sense to sort them
     // just for the sake of checking the panic message.
     #[should_panic]
-    fn should_not_panic_if_hashmap_does_not_contain_key() {
+    fn should_panic_if_hashmap_does_not_contain_key() {
```

Comment:
Renamed a couple tests where the name didn't seem to exactly match what the test was doing